rocks/RocksDatabase.java

```diff
@@ -143,21 +145,20 @@ static RocksDatabase loadAndOpen(RocksTypeDB typedb, String name, Factory.Sessio
     }
 
     protected void initialise() {
-        initialiseSchema();
+        initialiseOrLoadSchema();
```

Comment:
update naming to clarify the same method is used to initialise or load the schema rocks database